sql/core/src/main/scala/org/apache/spark/sql/catalyst/SQLBuilder.scala

```diff
@@ -219,6 +219,9 @@ class SQLBuilder private (
     case OneRowRelation =>
       ""
 
+    case p: View =>
```

Comment:
We will remove the entire rule after we have implemented the late binding method.